<center>
    <img src="https://cf-courses-data.s3.us.cloud-object-storage.appdomain.cloud/IBMDeveloperSkillsNetwork-DS0105EN-SkillsNetwork/labs/Module2/images/SN_web_lightmode.png" width="300" alt="cognitiveclass.ai logo">
</center>


#### Add your code below following the instructions given in the course


# My Jupyter Notebook on IBM Watson Studio

**Lucas Felipe Silva**  
Data Scientist

I am interested in data science because its important to understand the concepts of the actual world. 
Everytinh works around data, and those who does not use data are left behind.

<h3>I created a decorator called timer that measures how much time does the decorated function takes to run.<h3>

In [6]:
import time

def timer(func): 
    def wrapper():
        start = time.time()
        func()
        end = time.time()
        print(f"function {func.__name__} took {str(end - start)} seconds to run")
    return wrapper

@timer
def main():
    for n in range(1000000):
        pass
    time.sleep(2.14)
        
main()

function main took 2.1789207458496094 seconds to run


1. Item 1
2. Item 2 
---
* To-do List
  * Task 1
  * Task 2
    1. Task 2.1
    1. Task 2.2
---
```python
@timer
def main():
    for n in range(1000000):
        pass
    time.sleep(2.14)
```